MetadataExtractor/Formats/Apple/BplistReader.cs

```diff
@@ -36,17 +36,17 @@ public static bool IsValid(byte[] bplist)
         return true;
     }
 
-    public static PropertyListResults Parse(byte[] bplist)
+    public static PropertyListResults Parse(ReadOnlySpan<byte> bplist)
```

Comment:
General question: what do you think about passing `ReadOnlySpan<byte>` as an `in` parameter to avoid copies?